lib/faker/address/es.ex

```diff
@@ -429,37 +429,44 @@ defmodule Faker.Address.Es do
   end
 
   @doc """
-  Return state.
+  Return state. But Spain doesn't has states so this calls Faker.Address.Es.region() instead
+  """
+  @spec state() :: String.t()
+  def state, do: region()
+
+  @doc """
+  Return region.[Source](https://www.ine.es/daco/daco42/codmun/cod_ccaa.htm)
 
   ## Examples
 
-      iex> Faker.Address.Es.state()
-      "Castilla-La Mancha"
-      iex> Faker.Address.Es.state()
-      "Comunidad Valenciana"
-      iex> Faker.Address.Es.state()
-      "Galicia"
-      iex> Faker.Address.Es.state()
-      "Cataluña"
+      iex> Faker.Address.Es.region()
+      "Extremadura"
+      iex> Faker.Address.Es.region()
+      "Aragón"
```

Comment:
As a pattern we use 4 examples in the doctests.

Reply:
Thanks, I'll add them